applications/materialDataCreator/materialDataCreator.html

```diff
@@ -0,0 +1,573 @@
+            <table style='width:100%'><tr><td style='width: 15%; text-align: left;'><font class = 'Explain'>|&lt; Nonmetal</font></td><td style='width: 15%; text-align: left;'><font class = 'Explain'>| Hair</font></td><td style='width: 60%; text-align: left;'><font class = 'Explain'>| Chitin</font></td><td style='width: 10%; text-align: right;'><font class = 'Explain'>Metal &gt;|</font></td></tr></table>
+            <font class='label'>Metallic Map (Grayscale) URL:</font> <input name = 'metallicMap' type = 'text' style= 'width:100%;' oninput="save('metallicMap', this);">
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Roughness:</font> <input name = 'roughness_value' readonly type = 'text' size = '5'><br>
+            <input type="range"  style="width:100%;" min="0" max="1000" value="0" class="slider" name = 'roughness' id = 'roughness' oninput="save('roughness', this); document.materialEditor.roughness_value.value = (this.value/1000);"><br>
+            <table style='width:100%'><tr><td style='width: 50%; text-align: left;'><font class = 'Explain'>|&lt; Glossy, polished, lustrous</font></td><td style='width: 50%; text-align: right;'><font class = 'Explain'>Unpolished, mat, rough &gt;|</font></td></tr></table>
+            <font class='label'>Roughness Map (Grayscale) URL:</font> <input name = 'roughnessMap' type = 'text' style= 'width:100%;' oninput="save('roughnessMap', this);">
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Normal (bump) Map URL:</font> <input name = 'normalMap' type = 'text' style= 'width:100%;' oninput="save('normalMap', this);">
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Opacity:</font> <input name = 'opacity_value' readonly type = 'text' size = '5'><br>
+            <input type="range"  style="width:100%;" min="0" max="1000" value="0" class="slider" name = 'opacity' id = 'opacity' oninput="save('opacity', this); document.materialEditor.opacity_value.value = (document.materialEditor.opacity.value/1000);"><br>
+            <table style='width:100%'><tr><td style='width: 50%; text-align: left;'><font class = 'Explain'>|&lt; Transparent</font></td><td style='width: 50%; text-align: right;'><font class = 'Explain'>Opaque &gt;|</font></td></tr></table>
+            <font class='label'>Opacity Map: </font> <input type="radio" checked name = 'opacityMap' value = 'DO NOT USE' oninput="save('opacityMap', this);">Do not used&nbsp;&nbsp;&nbsp;
+            <input type="radio" name = 'opacityMap' value = 'USE' oninput="save('opacityMap', this);">Use<br>
+            <font class='Explain'>Not, for an opacity map, the alpha layer of the Albedo Map must be used. ( &lt; 2% and > 98% are suported, otherwise the avarage will be applied to all the surface. )</font>

```

Comment:
```suggestion
            <font class='Explain'>Not for an Opacity Map! The alpha layer of the Albedo Map must be used. ( &lt; 2% and > 98% are supported, otherwise the average will be applied to all the surface. )</font>
```

Reply:
Hmm... it was supposed to be this:
**Note: For an opacity map, the alpha layer of the Albedo Map must be used.**

Let me know if this makes more sense.

Reply:
I fixed "supported" and "average"